tests/integration/helpers.py

```diff
@@ -151,12 +151,44 @@ def project_exists(self, project_name):
         if location is _sentinel:
             location = self.example_repo
         repository_id = self.uuid4()
-        repository_iri = self.resource("repositories", repository_id)
+        repository_iri = self.resource("stacks", repository_id)
         await self.insert_node(Node(repository_iri, {
-            RDF.type: Doap.GitRepository,
+            RDF.type: Doap.Stack,
             Mu.uuid: repository_id,
             Doap.location: location,
         }))
+
+        drc_id = self.uuid4()
+        drc_text = """
+            version: \"2\"
+            services:
+
+              service1:
+                image: busybox
+                command: \"sleep 60\"
+
+              service2:
+                image: busybox
+                command: \"sleep 60\"
+        """
+        d_iri = IRI("http://stack-builder.big-data-europe.eu/resources/"
+        drc_iri = d_iri + "%s/%s" % ("docker-composes", drc_id)
+        drc_title = "stack_{}_drc_{}".format(repository_id, drc_id)
+        drc_node = Node(drc_iri, {
+            RDF.type: Stackbuilder.DockerCompose,
+            Mu.uuid: drc_id,
+            Dct.title: drc_title,
+            Stackbuilder.text: drc_text
```

Comment:
It's considered good practice in Python to insert trailing commas in multiline list and dict. Your habits in JS are visible to me 😀 But even in JS it becomes a standard: https://github.com/airbnb/javascript#commas--dangling

The reason we do that is to avoid touching irrelevant lines in a diff when the code change. If I add a new value in this dict at the end, I will be forced to add the comma and that will make the hunk of the diff more important than it should be. 

Example:

```patch
         drc_iri = d_iri + "%s/%s" % ("docker-composes", drc_id)
         drc_title = "stack_{}_drc_{}".format(repository_id, drc_id)
         drc_node = Node(drc_iri, {
             RDF.type: Stackbuilder.DockerCompose,
             Mu.uuid: drc_id,
             Dct.title: drc_title,
-            Stackbuilder.text: drc_text
+            Stackbuilder.text: drc_text,
+            Stackbuilder.newStuff: 42
         })
 
         await self.insert_triples([
             drc_node
         ])
```

Instead of:

```patch
         drc_iri = d_iri + "%s/%s" % ("docker-composes", drc_id)
         drc_title = "stack_{}_drc_{}".format(repository_id, drc_id)
         drc_node = Node(drc_iri, {
             RDF.type: Stackbuilder.DockerCompose,
             Mu.uuid: drc_id,
             Dct.title: drc_title,
             Stackbuilder.text: drc_text,
+            Stackbuilder.newStuff: 42
         })
 
         await self.insert_triples([
             drc_node
         ])
```